scio-bigquery/src/main/scala/com/spotify/scio/bigquery/types/ConverterProvider.scala

```diff
@@ -181,7 +176,7 @@ private[types] object ConverterProvider {
         case t if t =:= typeOf[Float]      => q"$s.toFloat"
         case t if t =:= typeOf[Double]     => q"$s.toDouble"
         case t if t =:= typeOf[String]     => q"$s"
-        case t if t =:= typeOf[BigDecimal] => bdecimal(tree)
+        case t if t =:= typeOf[BigDecimal] => q"BigDecimal($tree.toString)"
```

Comment:
I was looking into this and the precision here is  `34 >= p <= 38`. so I think it's better and always use the same precision. `38`